sources/T-REC-J.1108-201901-P.MSW-E.adoc

```diff
@@ -147,6 +157,7 @@ Therefore, the following requirements is required to perform the function of tim
 [TIME-REQ-11] The RoIP headend for RF signal reconstruction is required to transform Digital to Analogue and frequency band.
 
 [bibliography]
+[[bibliography]]
```

Comment:
@anermina we don't need an anchor for bibliography since it won't really be referred to.